Lexic.txt

```diff
@@ -0,0 +1,28 @@
+﻿Alphabet:
+* upper (A-Z) and lower case letters (a-z) of the English alphabet
+* decimal digits (0-9)
+
+
+1. Special symbols:
+* operators: +  -  *  /  =  ==  !=  !  >  <  >=  <=  %  ++  - -  ^
+* separators: () [ ] { } ;  :
+* res. words: if else while number string let const read print true false for
+
+
+2. Identifiers:
+* a sequence of letters and digits, such that the first character is a letter
+<id> :: = <letter> | <letter><seq_let_digit> 
+<seq_let_digit> :: = <letter><seq_let_digit> | <digit><seq_let_digit>

```

Comment:
You should also add the base case for the recursion, like <seq_let_digit> ::= <letter> | <letter><seq_let_digit> | <digit><seq_let_digit>